pyproject.toml

```diff
@@ -70,6 +70,10 @@ build-backend = "poetry.core.masonry.api"
 addopts = "--doctest-modules --strict-markers"
 filterwarnings = ["always::ImportWarning", "always::ResourceWarning"]
 testpaths = ["tests"]
+markers = [
+  # Defines custom markers used by measurement link python tests. Prevents PytestUnknownMarkWarning.  
+  "expect_discovery_service_stub_error: boolean specifying that the fake discovery service stub is valid or errored."
```

Comment:
I had trouble understanding this marker based on the name and description. It is not clear that this marker causes the fixture to return errors.

Reply:
Removed this marker since I have changed to direct test parameterization approach.